src/lib/storage/chunk.hpp

```diff
@@ -43,6 +43,7 @@ class Chunk : private Noncopyable {
 
  protected:
   // Implementation goes here
+  std::vector<std::shared_ptr<AbstractSegment>> _segments{};
```

Comment:
We're not quite sure we fully understand, would you be able to elaborate a little, if this point still stands?

We're under the impression, that member default initialization is recommended if the constructor doesn't do it (and even if it does just in case one misses it). And for initialization, we believe that the curly braces are recommended.